slippers/conf.py

```diff
@@ -9,6 +9,14 @@ def SLIPPERS_RUNTIME_TYPE_CHECKING(self) -> bool:
         """Enable runtime type checking of props"""
         return getattr(django_settings, "SLIPPERS_RUNTIME_TYPE_CHECKING", django_settings.DEBUG)  # type: ignore
 
+    @property
+    def SLIPPERS_CORE_COMPONENT_ATTRS(self) -> str:
+        """Core component allowed attributes
+
+        By default, return the common HTMX attributes & id
+        """
+        return getattr(django_settings, "SLIPPERS_CORE_COMPONENT_ATTRS", "id hx-get hx-post hx-on hx-push-url hx-select hx-select-oob hx-swap hx-swap-oob hx-target hx-trigger hx-vals")
```

Comment:
Very long line... we could break the line, and we could also add black and co to this repo.

Reply:
yep, good point. i'll create a story to add hooks
https://github.com/QuickRelease/slippers/pull/23/files#diff-194ca624ff8e097890eb9c81ab17f32a214113f419f3ebf8719bd8cd631245eeR21

Reply:
https://app.shortcut.com/quick-release/story/48159/slippers-add-pre-commit-hooks